src/olympia/versions/models.py

```diff
@@ -583,10 +594,7 @@ def inherit_nomination(sender, instance, **kw):
     if (instance.nomination is None and
             addon.status in amo.UNREVIEWED_ADDON_STATUSES and not
```

Comment:
Was this particular condition the cause of the problem? Since `inherit_nomination()` is called on `post_save`, it feels weird to keep that condition in there and manually call a different method to fix this issue.